src/powfacpy/plot_interface.py

```diff
@@ -33,22 +33,26 @@ def set_active_graphics_page(self,page):
 
 
   def set_active_plot(self,name_or_obj,graphics_page=None):
-    """Set the currently active plot.
+    """Set the currently active plot. Adjusts the active graphics
+    page accordingly if name_or_object is a PF object (the graphics
```

Comment:
what kind of PF object? can it be any or only plot object?

Reply:
Replaced with PF plot object